CSharp/BatchExplorer/Service/BatchService.cs

```diff
@@ -414,5 +449,26 @@ private void Dispose(bool disposing)
 
             this.disposed = true;
         }
+
+        private async Task<bool> WaitForResizingOperationToFinish(string poolId)
+        {
+            var pool = await this.GetPoolAsync(poolId);
+
+            int waitForSeconds = 10;
+            int count = 1;
+            while (count < waitForSeconds)
+            {
+                if (pool.AllocationState == AllocationState.Steady)

```

Comment:
💡 Just pointing it out, `{}` missing for if statement which is inconsistent with the other if in the file which has curly braces with 1 line inside if-statement.